R/derive_param_exist_flag.R

```diff
@@ -83,7 +83,7 @@
 #'
 #' @param by_vars Variables to uniquely identify a group
 #'
-#'   A list of symbols created using `exprs()` is expected.
+#'   `r roxygen_param_by_vars(unique = TRUE)`
```

Comment:
`unique` should not be specified as the grouping variables don't need to be a unique key.